packages/discovery/src/discovery/interactive/InteractiveDiffIgnore.ts

```diff
@@ -0,0 +1,196 @@
+import select from '@inquirer/select'
+import { ContractParameters } from '@l2beat/discovery-types'
+import chalk from 'chalk'
+
+import { DiscoveryOverridesBuilder } from '../config/DiscoveryOverridesBuilder'
+import { DiscoveryDiff } from '../output/diffDiscovery'
+
+interface Change {
+  contract: string
```

Comment:
maybe rename to `contractName` since I see that it clashes with some of the things